docker-compose.yml

```diff
@@ -1,9 +1,19 @@
 db:
   image: postgres
+  environment:
+    - POSTGRES_HOST_AUTH_METHOD=trust
 web:
   image: ccnmtl/mediathread
   environment:
-    - SETTINGS=settings_compose
+    - SETTINGS=settings_docker
+    - DB_HOST=db
+    - DB_USER=postgres
+    - DB_NAME=postgres
+    - DB_PORT=5432
+    - SECRET_KEY=CHANGEME
+    - EMAIL_BACKEND=django.core.mail.backends.dummy.EmailBackend
```

Comment:
Quick note on this setting - We should walk through how will email work in the production environment.